examples/image-streams/image-streams-centos7.json

```diff
@@ -1,5 +1,5 @@
 {
-  "kind": "List",
+  "kind": "ImageStreamList",
```

Comment:
It looks like this change is making FIPS fail

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27460/pull-ci-openshift-origin-master-e2e-aws-ovn-fips/1578481788341194752

Reply:
I don't know anything about this, but maybe the apiVersion should be image.openshift.io/v1? I see examples of that being used on ImageStreams (not many examples of ImageStreamList).